src/dispatch/plugins/dispatch_slack/models.py

```diff
@@ -29,3 +31,12 @@ class SubjectMetadata(BaseModel):
 
     project_id: Optional[str]
     channel_id: Optional[str]
+
+
+class CaseSubjects(DispatchEnum):
```

Comment:
@mvilanova @kevgliss prefer this pattern over the current raw str? I can add to everything.

Reply:
LGTM